plugins/plugin_utils/ansible_type.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2024 Vladimir Botka <vbotka@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from ansible.errors import AnsibleFilterError
+from ansible.module_utils.common._collections_compat import Mapping
+
+
+def _atype(data, alias):
+    """
+    Returns the name of the type class.
+    """
+
+    data_type = type(data).__name__
+    return alias.get(data_type, data_type)
+
+
+def _ansible_type(data, alias):
+    """
+    Returns the Ansible data type.
+    """
+
+    if not alias:
```

Comment:
```suggestion
    if alias is None:
```
Otherwise the user can pass `[]` or `""` and won't get an error that it should be a dictionary.